scripts/LTR_passage/append_pred.py

```diff
@@ -0,0 +1,53 @@
+'''Converts MSMARCO's tsv collection to Anserini jsonl files with field configurations.'''
```

Comment:
`LTR_passage` seems underspecified... this is just for MS MARCO passage, right?

So `msmarco_passage_ltr`?